### PR TITLE
V3 Docs

### PR DESCRIPTION
**Pull request by microstrain-sam**

### CODE REVIEW

File: src/cpp/microstrain/common/index.hpp [hunk 1]
```diff
@@ -70,7 +70,7 @@ namespace microstrain
     ////////////////////////////////////////////////////////////////////////////////
     ///@brief Representes an ID number ranging from 1..N including N.
     ///
-    /// This is interchangeable with the Index class above.
+    /// This is interchangeable with the Index class.
```
Comment (dacuster) [inline src/cpp/microstrain/common/index.hpp:73]: Maybe add `@ref` for Index?

File: src/cpp/mip/definitions/commands_3dm.hpp [hunk 2]
```diff
@@ -16,9 +16,9 @@ struct mip_interface;
 namespace commands_3dm {
 
 ////////////////////////////////////////////////////////////////////////////////
-///@addtogroup MipCommands_cpp  MIP Commands [CPP]
+///@addtogroup MipCommands_cpp
 ///@{
-///@defgroup 3dm_commands_cpp  3dm Commands [CPP]
+///@defgroup 3dm_commands_cpp  3dm Commands
```
Comment (dacuster) [inline src/cpp/mip/definitions/commands_3dm.hpp:21]: Same as C, 3DM vs 3dm

**Review by dacuster — APPROVED** [review]
Should be good once the comments are addressed. I'll pre-approve it now so it can just be merged

File: src/cpp/mip/definitions/common.hpp [hunk 3]
```diff
@@ -2,6 +2,19 @@
 
 #include <mip/mip_serialization.hpp>
 
+////////////////////////////////////////////////////////////////////////////////
+///@addtogroup mip_cpp
+///@{
+///
+///@defgroup MipCommands_cpp
```
Comment (dacuster) [inline src/cpp/mip/definitions/common.hpp:9]: Forgot the group title

File: src/cpp/mip/definitions/common.hpp [hunk 4]
```diff
@@ -2,6 +2,19 @@
...
+///@brief Contains all MIP command definitions.
+///
+///@defgroup MipData_cpp
```
Comment (dacuster) [inline src/cpp/mip/definitions/common.hpp:12]: Group title

**Review by dacuster — APPROVED** [review]
Forgot a couple more things

File: src/c/mip/definitions/commands_3dm.h [hunk 5]
```diff
@@ -17,9 +17,9 @@ extern "C" {
 #endif // __cplusplus
 
 ////////////////////////////////////////////////////////////////////////////////
-///@addtogroup MipCommands_c  MIP Commands [C]
+///@addtogroup MipCommands_c
 ///@{
-///@defgroup 3dm_commands_c  3dm Commands [C]
+///@defgroup 3dm_commands_c  3dm Commands
```
[the remark this answers is not in the document]
Comment (microstrain-sam) [reply src/c/mip/definitions/commands_3dm.h:22]: Unfortunately that's autogenerated code so it's hard to change that.

File: src/c/mip/mip_packet.h [hunk 6]
```diff
@@ -67,8 +67,8 @@ typedef struct mip_packet_view
 void mip_packet_create(mip_packet_view* packet, uint8_t* buffer, size_t buffer_size, uint8_t descriptor_set);
```
[the remark this answers is not in the document]
Comment (microstrain-sam) [reply src/c/mip/mip_packet.h:68]: This isn't an issue for documentation anymore. Leaving it as-is because it goes with the other init functions.

**Pull request merged**